lib/gateway.js

```diff
@@ -379,19 +379,27 @@ async function buildGateway (gatewayOpts, app) {
     serviceMap,
     entityResolversFactory: factory,
     pollingInterval: gatewayOpts.pollingInterval,
-    async refresh () {
+    async refresh (shouldRetry) {
```

Comment:
I don't understand the naming of this parameter

Reply:
It's a boolean parameter which specified if the retries should be done or not. should I change it to something like `isRetry`?